components/use-cases/ecommerce/ecommerce.md

```diff
@@ -0,0 +1,252 @@
+
+![Shopify Analytics to Keboola Provided Snowflake Database](/components/use-cases/ecommerce/shopify-to-keboola-snowflake.png)
+
+When you are finished, click **Save** in the top right corner. The template builder will create your new configuration and when it is done, you will be redirected to Use Case Catalogue and see the newly created flow. 
+
+Click :arrow_forward: **Run Use Case** and start building your visualizations a few minutes after. 
+
+![Shopify Analytics to Keboola Provided Snowflake Database - Flows](/components/use-cases/ecommerce/shopify-to-keboola-provided-snowflake-database-flows.png)
+
+## Authorizing eCommerce Data Sources
+To use a selected data source component, you must first authorize the data source. 
+
+At least one data source must be used in order to create a working flow.
+
+### Shopify Analytics
+
+![Shopify Source](/components/use-cases/ecommerce/shopify-source.png)
+
+To enable this application, you must:
+
+- [Enable private app development](https://help.shopify.com/en/manual/apps/private-apps#enable-private-app-development-from-the-shopify-admin) for your store.
+- Create a private application.
+- Enable `Read access` ADMIN API PERMISSIONS for the following following objects:
+    - `Orders`
+    - `Products`
+    - `Inventory`
+    - `Customers`
+
+Additional documentation is available [here](https://bitbucket.org/kds_consulting_team/kds-team.ex-shopify/src/master/README.md).
+
+### Shoptet Analytics
+
+This extractor allows you to download data from Shoptet permalinks. 
+
+![Shoptet Data Source](/components/use-cases/ecommerce/shoptet-source.png)
+
+Find all links in your Shoptet account. 
+
+Example: Go to Customers and click **Export**. At the bottom you will see a link looking like this: https://www.yourshopaddress.domain/export/customers.xml?ip=11.111.111.1111&hash=somehash
+
+- Set the shop name.
+- Set the Base URL of the store.
+- Set the Orders URL so that it contains the pattern ID and hash in the URL.
+- Set the Products URL so that it contains the pattern ID and hash in the URL.
+- Set the Customers URL so that it contains the hash in the URL.
+- Set the Stock URL so that it contains the hash in the URL.
+
+### WooCommerce
+
+To download data form [WooCommerce](https://bitbucket.org/kds_consulting_team/kds-team.ex-woocommerce/src/master/README.md) we need to configure:
+
+- Store_url: Website Domain name where WooCommerce is hosted. e.g. https://myshop.com
+
+- consumer_key: Rest API Consumer Key from WooCommerce Admin panel
+
+- consumer_secret: Rest API Consumer Secret from WooCommerce Admin panel
+
+- date_from: Inclusive Date in YYYY-MM-DD format or a string i.e. 5 days ago, 1 month ago, yesterday, etc.
+
+- date_to: Exclusive Date in YYYY-MM-DD format or a string i.e. 5 days ago, 1 month ago, yesterday, etc.
```

Comment:
toto jsou hrozně obskurní instrukce :)

Reply:
@MonikaFeigler

Reply:
ok, souhlasím

Reply:
woocommerce Hnača skryla, až to bude aktuální, napíšu to lépe